src/jit/codegen.h

```diff
@@ -465,11 +465,11 @@ protected :
                                     IL_OFFSETX                                    ilOffset);
 
 
+#if defined(_TARGET_ARM_)
     //
```

Comment:
jit.h

---
In reply to: [72658054](https://github.com/dotnet/coreclr/pull/6491#discussion_r72658054) [](ancestors = 72658054)